Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -55,8 +58,21 @@ public struct PreplannedListItemView: View {
     @ViewBuilder private var downloadButton: some View {
         switch model.status {
         case .downloaded:
-            Image(systemName: "checkmark.circle")
-                .foregroundStyle(.secondary)
+            Button {
+                Task {
+                    if let map = await model.loadMobileMapPackage() {
+                        onMapSelectionChanged?(map)
+                    }
+                }
+            } label: {
+                Text("Open")
+                    .font(.system(.headline, weight: .bold))
+                    .foregroundColor(.accentColor)
```

Comment:
`foregroundColor` is [deprecated](https://developer.apple.com/documentation/swiftui/view/foregroundcolor(_:)). Can you log an GitHub issue to clean them up later? Thanks.
```suggestion
                    .foregroundStyle(Color.accentColor)
```

Reply:
Logged an [issue](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/issues/842).